expression/builtin_arithmetic.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/chunk"
-	math2 "github.com/pingcap/tidb/util/math"
+	math2 "github.com/pingcap/tidb/util/mathutil"
```

Comment:
Please remove the alias and github.com/cznic/mathutil dependency and use the generics from github.com/pingcap/tidb/util/mathutil instead.

Reply:
Done